src/darwin/Framework/CHIP/MTRCluster.h

```diff
@@ -28,6 +28,9 @@ typedef void (^SubscriptionEstablishedHandler)(void);
 typedef void (^MTRStatusCompletion)(NSError * _Nullable error);
 typedef void (^MTRSubscriptionEstablishedHandler)(void);
 
+MTR_PROVISIONALLY_AVAILABLE
+typedef void (^MTRDiagnosticLogsRetrievedCompletion)(NSError * _Nullable error, NSArray<NSString *> * _Nullable filePathArray);
```

Comment:
If we use it in only one place, we probably shouldn't define a typedef for this thing at all; just have the type inlined in the method decl.

Reply:
done.